### ISSUE TITLE
Request - Hide Silver and Copper

### ISSUE THREAD

**Issue by pqee** [wow-addon/Broker_CurrencyFlow#9]
Any chance you could have an option to round to the nearest gold, or just hide the silver/copper?  Or make it auto hide once at max level?

**Comment by tflo** [comment]
Yeah, something like the existing Short form, but _without_ the Silver decimal, and _with_  thousands delimiters:

Current Short form: `1234567.8`
Better: `1,234,567` 

(The Silver decimal could optionally be added for  amounts <100g, but for everything else it's rather pointless, IMHO.)
